packages/kbn-lens-embeddable-utils/attribute_builder/visualization_types/layers/xy_data_layer.ts

```diff
@@ -88,6 +91,9 @@ export class XYDataLayer implements ChartLayer<XYDataLayerConfig> {
             options: {
               ...options.buckets.params,
               sourceField: options.buckets.field ?? dataView.timeFieldName,
+              params: {
+                interval: this.layerConfig.options?.params?.interval ?? 'auto',
+              },
```

Comment:
`options.buckets.params` already contains the `interval` info. We actually need to change to the `getHistogramColumn` as it already knows how to construct the histogram object. 


```ts
export const getHistogramColumn = ({
  columnName,
  options,
}: {
  columnName: string;
  options?: Partial<
    Pick<DateHistogramIndexPatternColumn, 'sourceField'> & {
      params: DateHistogramColumnParams;
    }
  >;
}) => {
  const { interval, ...rest } = options?.params ?? {};
  return {
    [columnName]: {
      dataType: 'date',
      isBucketed: true,
      label: '@timestamp',
      operationType: 'date_histogram',
      scale: 'interval',
      sourceField: '@timestamp',
      ...rest,
      params: { interval: interval ?? 'auto', ...options?.params },
    } as DateHistogramIndexPatternColumn,
  };
};
```

After changing the `getHistogramColumn` function, you can remove these lines